cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1028,6 +1028,14 @@ public static final class JVMResource {
       public static final String THREAD_COUNT = "jvm.resource.thread.count";
     }
 
+    /**
+     * Program event publish
+     */
+    public static final class ProgramEvent {
+      public static final String PUBLISHED_COUNT = "program.event.published.count";
+      public static final String METRICS_FETCH_LATENCY_MS = "program.event.metrics.fetch.latency.millis";
```

Comment:
nit: same here, maybe add `spark` in the constant somewhere? (unless we anticipate in the future that this might not only consist of spark metrics)

Reply:
Done